### PR TITLE
Add Identifier handler

### PR DESCRIPTION
**Pull request by senthalan**
merge after: https://github.com/wso2/carbon-identity-framework/pull/1824

### CODE REVIEW

File: components/org.wso2.carbon.identity.application.authentication.handler.identifier/src/main/java/org/wso2/carbon/identity/application/authentication/handler/identifier/internal/IdentifierAuthenticatorServiceComponent.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+import org.wso2.carbon.identity.application.authentication.framework.ApplicationAuthenticator;
+import org.wso2.carbon.identity.application.authentication.handler.identifier.IdentifierHandler;
+import org.wso2.carbon.user.core.service.RealmService;
+
+/**
+ * @scr.component name="identity.application.handler.identifier.component" immediate="true"
+ * @scr.reference name="realm.service"
+ * interface="org.wso2.carbon.user.core.service.RealmService"cardinality="1..1"
+ * policy="dynamic" bind="setRealmService" unbind="unsetRealmService"
+ */
```
Comment (thanujalk) [inline components/org.wso2.carbon.identity.application.authentication.handler.identifier/src/main/java/org/wso2/carbon/identity/application/authentication/handler/identifier/internal/IdentifierAuthenticatorServiceComponent.java]: can we have the new annotations for this?

File: components/org.wso2.carbon.identity.application.handler.identifier/src/main/java/org/wso2/carbon/identity/application/handler/identifier/internal/IdentifierAuthenticatorServiceComponent.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+    private static Log log = LogFactory.getLog(IdentifierAuthenticatorServiceComponent.class);
+
+    private static RealmService realmService;
+
+    public static RealmService getRealmService() {
+        return realmService;
+    }
+
+    protected void setRealmService(RealmService realmService) {
```
Comment (thanujalk) [inline components/org.wso2.carbon.identity.application.handler.identifier/src/main/java/org/wso2/carbon/identity/application/handler/identifier/internal/IdentifierAuthenticatorServiceComponent.java]: add an empty line here.

File: components/org.wso2.carbon.identity.application.authentication.handler.identifier/src/main/java/org/wso2/carbon/identity/application/authentication/handler/identifier/internal/IdentifierAuthenticatorServiceComponent.java [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+    private static Log log = LogFactory.getLog(IdentifierAuthenticatorServiceComponent.class);
+
+    private static RealmService realmService;
+
+    public static RealmService getRealmService() {
+        return realmService;
+    }
+
+    protected void setRealmService(RealmService realmService) {
```
Comment (thanujalk) [inline components/org.wso2.carbon.identity.application.authentication.handler.identifier/src/main/java/org/wso2/carbon/identity/application/authentication/handler/identifier/internal/IdentifierAuthenticatorServiceComponent.java]: add an empty line here.

File: components/org.wso2.carbon.identity.application.handler.identifier/src/main/java/org/wso2/carbon/identity/application/handler/identifier/internal/IdentifierAuthenticatorServiceComponent.java (hunk 2 above)
Comment (senthalan) [reply components/org.wso2.carbon.identity.application.handler.identifier/src/main/java/org/wso2/carbon/identity/application/handler/identifier/internal/IdentifierAuthenticatorServiceComponent.java]: fixed

File: components/org.wso2.carbon.identity.application.authentication.handler.identifier/src/main/java/org/wso2/carbon/identity/application/authentication/handler/identifier/internal/IdentifierAuthenticatorServiceComponent.java (hunk 1 above)
Comment (senthalan) [reply components/org.wso2.carbon.identity.application.authentication.handler.identifier/src/main/java/org/wso2/carbon/identity/application/authentication/handler/identifier/internal/IdentifierAuthenticatorServiceComponent.java]: changed

File: components/org.wso2.carbon.identity.application.authentication.handler.identifier/src/main/java/org/wso2/carbon/identity/application/authentication/handler/identifier/internal/IdentifierAuthenticatorServiceComponent.java (hunk 3 above)
Comment (senthalan) [reply components/org.wso2.carbon.identity.application.authentication.handler.identifier/src/main/java/org/wso2/carbon/identity/application/authentication/handler/identifier/internal/IdentifierAuthenticatorServiceComponent.java]: added

**Pull request merged**